src/components/board/Board.vue

```diff
@@ -23,26 +23,29 @@
 <template>
 	<div class="board-wrapper">
 		<Controls :board="board" />
-		<div v-if="loading" class="emptycontent">
-			<div class="icon icon-loading" />
-			<h2>{{ t('deck', 'Loading board') }}</h2>
-			<p />
-		</div>
-		<div v-else-if="board" class="board">
-			<Container lock-axix="y"
-				orientation="horizontal"
-				:drag-handle-selector="dragHandleSelector"
-				@drop="onDropStack">
-				<Draggable v-for="stack in stacksByBoard" :key="stack.id">
-					<Stack :stack="stack" />
-				</Draggable>
-			</Container>
-		</div>
-		<div v-else class="emptycontent">
-			<div class="icon icon-deck" />
-			<h2>{{ t('deck', 'Board not found') }}</h2>
-			<p />
-		</div>
+		<transition name="fade" mode="out-in">
+			<div v-if="loading" class="emptycontent" key="loading">
```

Comment:
key? Are they not only needed in transition group?

Reply:
> When toggling between elements that have the same tag name, you must tell Vue that they are distinct elements by giving them unique key attributes

https://vuejs.org/v2/guide/transitions.html#Transitioning-Between-Elements